modules/runner/src/integrationTest/java/org/apache/ignite/internal/distribution/zones/ItDistributionZonesFilterTest.java

```diff
@@ -298,7 +295,16 @@ void testEmptyDataNodesDoNotPropagatedToStableAfterAlteringFilter() throws Excep
 
         waitDataNodeAndListenersAreHandled(metaStorageManager, 2);
 
-        assertPendingStableAreNull(metaStorageManager, partId);
+        // TODO: https://issues.apache.org/jira/browse/IGNITE-19425 here we should have no nodes,
+        // which pass the filter, when dataNodes from DistributionZoneManager will be used
+        assertValueInStorage(
```

Comment:
why not `assertPendingWasNeverExsists`?

Reply:
The name and semantic of the test is testEmptyDataNodesDoNotPropagatedToStableAfterAlteringFilter, so, I thought it will be strange to check the pendings in the end

Reply:
Yes, you're right